lago/utils.py

```diff
@@ -97,13 +97,13 @@ def join_all(self, raise_exceptions=True):
 def invoke_in_parallel(func, *args_sequences):
     vt = VectorThread(func_vector(func, zip(*args_sequences)))
     vt.start_all()
-    vt.join_all()
+    return vt.join_all()
```

Comment:
just wondering - why is this change needed?

need to be careful here - OST uses this directly I think.